tslint.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I suppose it should be bought in as a dev dependency, but for now, I was just installing it globally.